ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -74,6 +74,8 @@ type structure_rule = {
     "not" ? boolean
     @{routing table action}
     "table" ? network_valid_routing_table
+    @{priroty also known as metric, needed by nm when setting routing rules)}
+    "priority" ? long
```

Comment:
This should at least be a positive integer, experimentally it appears to be limited to 32-bits.
```suggestion
    "priority" ? long(0..0xffffffff)
```